README.rst

```diff
@@ -1,8 +1,8 @@
-.. image:: https://travis-ci.org/Yelp/swagger-py.png?branch=master
-  :target: https://travis-ci.org/Yelp/swagger-py?branch=master
+.. image:: https://travis-ci.org/Yelp/bravado.png?branch=master
+  :target: https://travis-ci.org/Yelp/bravado?branch=master
 
 
-Swagger-py
+Bravado
 ==========
```

Comment:
Very minor: might consider deleting the extra `=` header chars here and elsewhere now that we're using a shorter name.